### PR TITLE
kubeadm: De-dup the confirmation on the interactive cmds

### PR DESCRIPTION
**Pull request by chendave**
Signed-off-by: Dave Chen <dave.chen@arm.com>

#### What type of PR is this?

/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
Originally, `kubeadm reset` only accept the 'y' when the force flag is provided, now it accepts the 'yes' as what `kubeadm upgrade apply` does.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

/sig cluster-lifecycle

### CODE REVIEW

File: cmd/kubeadm/app/cmd/upgrade/apply.go [hunk 1]
```diff
@@ -30,6 +31,7 @@ import (
 
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/options"
+	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
```
Comment (chendave) [inline cmd/kubeadm/app/cmd/upgrade/apply.go:34]: kubeadm now has two `util` packages, not sure whether we can merge them.

`"k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"`
`"k8s.io/kubernetes/cmd/kubeadm/app/util"`

**Comment by chendave** [comment]
unrelated race issue.

/retest

File: cmd/kubeadm/app/cmd/upgrade/apply.go (hunk 1 above)
Comment (neolit123) [reply cmd/kubeadm/app/cmd/upgrade/apply.go:34]: Cmd / backend util separation is usually good to have. I have not looked at the current state but we can keep them for now.

**Review by neolit123 — COMMENTED** [review]
Thanks
/approve
/triage accepted

File: cmd/kubeadm/app/cmd/upgrade/apply.go (hunk 1 above)
Comment (chendave) [reply cmd/kubeadm/app/cmd/upgrade/apply.go:34]: thanks for the reply!

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Pull request merged**